build/images/scripts/start_ovs_windows_containerd.ps1

```diff
@@ -0,0 +1,146 @@
+$ImportCertificate = $true
+$CheckFileHash = $true
+$PowerShellModuleBase = "C:\Windows\System32\WindowsPowerShell\v1.0\Modules"
+$InstallLog = "$OVSDownloadDir\install_ovs.log"
+
+function WaitExpandFiles($Src, $Dest) {
+    Write-Host "Extract $Src to $Dest"
+    Expand-Archive -Path $Src -DestinationPath $Dest | Out-Null
+}
+
+function CheckIfOVSInstalled() {
+    if (Test-Path -Path $OVSInstallDir) {
+        Write-Host "$OVSInstallDir already exists, OVS is already installed."
+        return $true
+    }
+    else {
+        Write-Host "$OVSInstallDir does not exist, OVS is not installed."
+        return $false
+    }
+}
+
+
+
+function DownloadOVS() {
+    If (!(Test-Path $OVSDownloadDir)) {
+        mkdir -p $OVSDownloadDir
+    }
+    Write-Host "Downloading OVS package from $OVSDownloadURL to $OVSZip"
+    curl.exe -sLo $OVSZip $OVSDownloadURL
+    If (!$?) {
+        Write-Host "Download OVS failed, URL: $OVSDownloadURL"
+        exit 1
+    }
+    if ($CheckFileHash) {
+        $FileHash = Get-FileHash $OVSZip
+        If ($OVSPublishedHash -ne "" -And $FileHash.Hash -ne $OVSPublishedHash) {
+            Write-Host "SHA256 mismatch for OVS download"
+            exit 1
+        }
+    }
+
+    Write-Host "Download OVS package success."
+}
+
+function InstallOVS() {
+    WaitExpandFiles $OVSZip $OVSDownloadDir
+    # Copy OVS package to target dir.
+    Write-Host "Copying OVS package from $OVSDownloadDir\openvswitch to $OVSInstallDir"
+    mv "$OVSDownloadDir\openvswitch" $OVSInstallDir
+    rm $OVSZip
+    # Create log and run dir.
+    $OVS_LOG_PATH = $OVSInstallDir + "\var\log\openvswitch"
+    If (!(Test-Path $OVS_LOG_PATH )) {
+        mkdir -p $OVS_LOG_PATH 
+    }
+    $OVSRunDir = $OVSInstallDir + "\var\run\openvswitch"
+    If (!(Test-Path $OVSRunDir)) {
+        mkdir -p $OVSRunDir 
+    }  
+}
+
+function InstallDriver(){
+   $OVSDriverDir = "$OVSInstallDir\driver"
+   # Install OVS driver certificate.
+   $DriverFile="$OVSDriverDir\OVSExt.sys"
+   if ($ImportCertificate) {
+       $CertificateFile = "$OVSDriverDir\package.cer"
+       if (!(Test-Path $CertificateFile)) {
+            $ExportType = [System.Security.Cryptography.X509Certificates.X509ContentType]::Cert;
+            $Cert = (Get-AuthenticodeSignature $DriverFile).SignerCertificate;
+            [System.IO.File]::WriteAllBytes($CertificateFile, $Cert.Export($ExportType));
+    }
+       Write-Host "Installing OVS driver certificate."
+       Import-Certificate -FilePath "$CertificateFile" -CertStoreLocation cert:\LocalMachine\TrustedPublisher
+       Import-Certificate -FilePath "$CertificateFile" -CertStoreLocation cert:\LocalMachine\Root
+   }
+ 
+   # Install OVS kernel driver.
+   Write-Host "Installing OVS kernel driver"
+   $VMMSStatus = $(Get-Service vmms -ErrorAction SilentlyContinue).Status
+   if (!$VMMSStatus) {
+       $VMMSStatus = "not exist"
+   }
+   Write-Host "Hyper-V Virtual Machine Management service status: $VMMSStatus"
```

Comment:
These operations are not required. We only need to use the `netcfg` command during the `initContainer` startup phase to check if the driver exists. If it already exists, we can skip the relevant installation steps. Otherwise, we can proceed with the driver installation using `netcfg -l`.